### PR TITLE
fixed help request api

### PR DESCRIPTION
**Pull request by ThomasRoehl**
Small fix for the "assistify.helpDiscussion" RestAPI endpoint.

+ Fixed API endpoint
+ Added roomID to the response body
+ Added 2 files to the .gitignore, which have to be added anyway

Example http request:
```
curl -H "X-Auth-Token: 9HqLlyZOugoStsXCUfD_0YdwnNnunAJF8V47U3QHXSq" \
     -H "X-User-Id: aobEdbYhXfu5hkeqG" \
     -H "Content-type:application/json" \
     http://localhost:3000/api/v1/assistify.helpDiscussion \
     -d '{ "support_area": "test", "seeker": {"id": "rocketchat.internal.admin.test", "email": "rocketchat.internal.admin.test@rocket.chat"}, "providers": [{"id": "rocketchat.internal.admin.test", "email": "rocketchat.internal.admin.test@rocket.chat"}] }'
```

Example response:
```
{
  "success": true,
  "url": "http://localhost:3000/request/Assistify_57",
  "id": "XfaXfftD2j4SytxrE",
  "members": [
    {
      "username": "rocketchat.internal.admin.test"
    }
  ]
}
```

### CODE REVIEW

File: packages/assistify-help-request/server/api.js [hunk 1]
```diff
@@ -151,6 +152,7 @@ export class HelpRequestApi {
 		const roomCreated = RocketChat.models.Rooms.findOne({_id: channel.rid});
 		return {
 			room: roomCreated,
+			_id: channel.rid,
```
Comment (mrsimpson) [inline packages/assistify-help-request/server/api.js]: `room._id` === channel.rid, or not?

File: packages/assistify-help-request/server/types.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 // Definition of value objects. No clue why export interface is not supported
 export class HelpDiscussionCreatedResponse {
-	constructor(url, members) {
+	constructor(url, id, members) {
 		this.success = true;
 		this.url = url;
+		this.id = id;
```
Comment (mrsimpson) [inline packages/assistify-help-request/server/types.js]: I'd prefer returning the room, any concerns?

File: .gitignore [hunk 3]
```diff
@@ -73,3 +73,5 @@ pm2.json
 settings.json
 build.sh
 /public/livechat
+.screenshots
+packages/rocketchat-google-vision/.npm/package/npm-shrinkwrap.json
```
Comment (mrsimpson) [inline .gitignore]: does that actually work? I tried that earlier but it was still appearing in the diff

**Comment by ThomasRoehl** [comment]
Changes made, as requested. New result looks like this:
```
{
  "success": true,
  "url": "http://localhost:3000/request/Assistify_10",
  "room": {
    "_id": "PjEGou2DzyYTYAzGL",
    "name": "Assistify_10",
    "fname": "Assistify_10",
    "t": "r",
    "msgs": 1,
    "u": {
      "_id": "rocketchat.internal.admin.test",
      "username": "rocketchat.internal.admin.test"
    },
    "expertise": "",
    "ts": "2018-01-28T16:40:19.158Z",
    "ro": false,
    "sysMes": true,
    "_updatedAt": "2018-01-28T16:40:19.247Z",
    "meta": {
      "revision": 3,
      "created": 1517157619164,
      "version": 0,
      "updated": 1517157619252
    },
    "$loki": 13,
    "usernames": [
      "rocketchat.internal.admin.test"
    ],
    "topic": "",
    "helpRequestId": "oGE8Amk6dZb3CF9QH",
    "v": {
      "_id": "rocketchat.internal.admin.test",
      "username": "rocketchat.internal.admin.test"
    }
  },
  "members": [
    {
      "username": "rocketchat.internal.admin.test"
    }
  ]
}
```

File: .gitignore (hunk 3 above)
Comment (ThomasRoehl) [reply .gitignore]: the .screenshots seems to work the second one, unfortunately, doesn't

**Comment by mrsimpson** [comment]
@ThomasRoehl 
as discussed, pls. reduce to

```javascript
{
  "success": true,
  "url": "http://localhost:3000/request/Assistify_10",
  "room": {
    "_id": "PjEGou2DzyYTYAzGL",
    "name": "Assistify_10",
    "t": "r",
    "expertise": "",
    "topic": "",
    "helpRequestId": "oGE8Amk6dZb3CF9QH"
  },
  "members": [
    {
      "username": "rocketchat.internal.admin.test"
    }
  ]
}
```

**Comment by ThomasRoehl** [comment]
Changes made, as requested!

**Pull request merged**